common-lib/src/main/scala/com/gu/mediaservice/lib/logging/MarkerUtils.scala

```diff
@@ -0,0 +1,16 @@
+package com.gu.mediaservice.lib.logging
+
+import net.logstash.logback.marker.LogstashMarker
+import net.logstash.logback.marker.Markers.appendEntries
+import scala.collection.JavaConverters._
+
+trait LogMarker { def toLogMarker: LogstashMarker }
+
+case class MarkerMap(values: Map[String, Any]) extends LogMarker {
+  override def toLogMarker: LogstashMarker = appendEntries(values.asJava)
+}
+
+trait MarkerUtils {
+  val FALLBACK: String = "unknown"
```

Comment:
Just out of interest (I can see it's not unique to this PR) – what is this for?

Reply:
It's playing the role of a constant - when creating makers from `Option`s we need to `getOrElse`, this is the `orElse` value, keeping things DRY.